app/domain/pbs/export/tabular/people/participation_nds_row.rb

```diff
@@ -9,11 +9,11 @@ module Pbs::Export::Tabular::People::ParticipationNdsRow
   extend ActiveSupport::Concern
 
   included do
-    alias_method_chain :first_language, :correspondence_language
+    alias_method_chain :first_language, :language
   end
 
-  def first_language_with_correspondence_language
-    lang = entry.correspondence_language.presence
-    lang ? lang.upcase : 'DE'
+  # TBD: move to youth wagon? https://github.com/hitobito/hitobito_youth/blob/3322054e2d64db19f8b049773e61f68e7f614546/app/domain/export/tabular/people/participation_nds_row.rb#L70
```

Comment:
@carlobeltrame Das könnte meiner Meinung nach in den youth-Wagon übernommen werden. https://github.com/hitobito/hitobito_youth/blob/3322054e2d64db19f8b049773e61f68e7f614546/app/domain/export/tabular/people/participation_nds_row.rb#L70

Reply:
Ja, finde ich auch, aber dann würde ich die Logik noch restriktiver schreiben. In der [Spezifikation von J+S](https://www.jugendundsport.ch/content/jus-internet/de/themen/neue-nds/datenimport/_jcr_content/contentPar/downloadlist_642662863/downloadItems/114_1649824850841.download/Personenimport_JA_de.pdf) sind nur DE, FR, IT und Andere als Beispielwerte aufgeführt. Wenn wir das in den Youth Wagon verschieben gibt es potenziell noch mehr Sprachen (z.B. bei der Stiftung junger Auslandschweizer) die da in Zukunft mal zu ungültigen NDS-Exports führen könnten. Also wenn verschieben, dann lieber in einem Mapping nachschauen, und auf "Andere" zurückfallen wenns nicht de, fr oder it ist.